packages/react-native-client/android/src/main/java/org/membraneframework/reactnative/FishjamForegroundService.kt

```diff
@@ -0,0 +1,90 @@
+package org.membraneframework.reactnative
```

Comment:
TODO: we should remove membraneframework namespace at some point...

Reply:
[FCE-528](https://linear.app/swmansion/issue/FCE-528)